src/components/pagination/_pagination.scss

```diff
@@ -289,6 +311,10 @@ $css--helpers: true;
     margin-left: $spacing-md;
     color: $text-02;
   }
+  .#{$prefix}--pagination__text[for='select-id-pagination-page'] {
```

Comment:
Will this selector apply if the `id` is changed? I don't know if we can use this reliably since there could be multiple pagination components on the page with different `id`s

Reply:
@joshblack FYI for context, this is specifically targeting a `label`, perhaps i could prefix the selector the the tag to be more explicit

this is potentially a problem with multiple pagination components… but i don't think the ID would over change? one alternative would be to use the selector `.#{$prefix}--pagination__right > .#{$prefix}--pagination__text`… thoughts?

Reply:
> but i don't think the ID would over change

We would need the IDs to change between inputs since they're supposed to be globally unique, I think 👍 That selector seems to work!